bedrock/newsletter/templates/newsletter/existing.html

```diff
@@ -15,20 +15,39 @@
 
   {% block main_feature %}
     <div id="main-feature">
-      <h1>{{ _('Manage your Email Preferences') }}</h1>
+    {% if did_confirm == '1' %}
```

Comment:
With my suggested change below, this would just be `{% if did_confirm %}`. It's just a bit safer to avoid sending user input (from the url) straight to the template context.